src/components/AllArticles/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import ArticleCard from './ArticleCard';
+
+const allArticlesUrl = 'https://valinor-ah-backend-stagi-pr-80.herokuapp.com/api/v1/articles/category/all';
```

Comment:
The base URL should be in an environment variable

Reply:
@mbilesanmi  I've fixed this now